backend/pharmacy/api/views.py

```diff
@@ -20,42 +28,74 @@ def get_token(cls, user):
 
         return token
 
+
 class MyTokenObtainPairView(TokenObtainPairView):
     serializer_class = MyTokenObtainPairSerializer
 
 
-@api_view(['GET'])
+# A method decorator to cache the view for 60 seconds
+@method_decorator(cache_page(60), name='get')
+# A method decorator to vary on the cookie
+@method_decorator(vary_on_cookie, name='get')
+# A method decorator to vary on the headers
+# @method_decorator(vary_on_headers, name='get')
+# Allow only authenticated users to access this view
 @permission_classes([IsAuthenticated])
```

Comment:
This will ensure that only authenticated user can access these api methods..... i.e. only if user is logged in